airbyte-cdk/python/airbyte_cdk/sources/declarative/interpolation/interpolated_string.py

```diff
@@ -28,6 +28,7 @@ def __post_init__(self, parameters: Mapping[str, Any]) -> None:
         self.default = self.default or self.string
         self._interpolation = JinjaInterpolation()
         self._parameters = parameters
+        self._just_string = None
```

Comment:
```suggestion
        self._is_plain_string = None
```

Naming doesn't sit right with me — something like this perhaps? Up to @artem1205 and @girarda though.